nevergrad/functions/control/core.py

```diff
@@ -68,6 +69,14 @@ def state_std(self):
     @property
     def policy_dim(self):
         raise NotImplementedError
+        
+    # pylint: disable=arguments-differ
+    def evaluation_function(self, x: np.ndarray) -> float:  # type: ignore
+        # pylint: disable=not-callable
+        loss = self.function(x)
```

Comment:
is there no noise in mujoco? if there is noise, it may be worth averaging over several runs.

Reply:
The noise is not merged, and I do not expect it to be merged in the near future.

Reply:
I mean, when there is no added noise, the behavior of the function is deterministic then? If yes that's fine and you can merge then

Reply:
Yes!